doc/life_of_a_language_feature.md

```diff
@@ -55,57 +59,164 @@ significant changes.
 
 If consensus is reached on a specific proposal and we decide to accept it, a
 member of the language team will be chosen to shepherd the implementation.
-The implementation will be tracked via three artifacts:
-
-  - An implementation plan document
+The implementation will be tracked via two artifacts:
 
   - A 'implementation' issue.
 
   - A feature specification document
 
-The implementation plan must be located in a sub-directory with the name of the
-feature (`/[feature-name]/`) located inside the `accepted/future-releases/`
-folder. The filename should be `implementation-plan.md`. The implementation
-plan should generally include at least:
-
-  - Affected implementation teams.
-
-  - Schedule and release milestones.
-
-  - Release flag if required, plan for shipping.
-
 The 'feature specification' is **a single canonical writeup of the language
 feature** which is to serve as the implementation reference. Feature
 specifications use Markdown format. The file name should be
-`feature-specification.md`, and the feature specification should
-be located in the same sub-directory as the implementation plan.
-
-A meta-issue (labelled `implementation`) will be filed in the language
-repository for tracking the implementation process. This top of this issue must
-contain links to:
+`feature-specification.md`, and the feature specification should be located
+inside the `accepted/future-releases/` folder.
 
-  - The related `request` issue
+The implementation issue (labelled `implementation`) will be filed in the
+language repository for tracking the implementation process. This top of this
+issue must contain links to:
 
-  - The related `feature` issue
-
-  - A link to the implementation plan
+  - The related `request` and `feature` issues (if they exist)
 
   - A link to the feature specification
 
+### Kick-off meetings
+
 The next step of the implementation issue is to get sign-off from all of the
 relevant implementation teams indicating that they understand the proposal,
 believe that it can reasonably be implemented, and feel that they have
-sufficient details to proceed.  There may be further iteration on the proposal
+sufficient details to proceed.
+
+At a minimum, the set of relevant implementation teams should nearly always
```

Comment:
I really appreciate the explanation behind this. It's the main reason why we want to emphasize kick-off meetings in the first place.

Could we perhaps write the teams in a way that looks more like a checklist/list? I think that'd be useful to look at when you're thinking of the teams you need to talk to.

Reply:
Sounds reasonable. Done.